src/bin/vip-dev-env-update.js

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+import debugLib from 'debug';
+import chalk from 'chalk';
+
+/**
+ * Internal dependencies
+ */
+import command from 'lib/cli/command';
+import { getEnvironmentName } from 'lib/dev-environment/dev-environment-cli';
+import { DEV_ENVIRONMENT_FULL_COMMAND } from 'lib/constants/dev-environment';
+import { addDevEnvConfigurationOptions, handleCLIException, promptForArguments } from '../lib/dev-environment/dev-environment-cli';
+import type { InstanceOptions } from '../lib/dev-environment/types';
+import { doesEnvironmentExist, readEnvironmentData, updateEnvironment } from '../lib/dev-environment/dev-environment-core';
+import { DEV_ENVIRONMENT_NOT_FOUND } from '../lib/constants/dev-environment';
+
+const debug = debugLib( '@automattic/vip:bin:dev-environment' );
+
+const examples = [
+	{
+		usage: `${ DEV_ENVIRONMENT_FULL_COMMAND } update`,
+		description: 'Retriggers setup wizard in order to change environment configuration',
+	},
+];
+const cmd = command().option( 'slug', 'Custom name of the dev environment' );
+
+addDevEnvConfigurationOptions( cmd );
+
+cmd.examples( examples );
+cmd.argv( process.argv, async ( arg, opt ) => {
+	const slug = getEnvironmentName( opt );
+
+	try {
+		const environmentAlreadyExists = doesEnvironmentExist( slug );
+		if ( ! environmentAlreadyExists ) {
+			throw new Error( DEV_ENVIRONMENT_NOT_FOUND );
+		}
+
+		const currentInstanceData = readEnvironmentData( slug );
+
+		debug( 'Read instance data', currentInstanceData );
+
+		const preselectedOptions = {
+			// Title and multisite can't be changed during update
+			title: currentInstanceData.wpTitle,
+			multisite: currentInstanceData.multisite,
+			...opt,
+		};
+
+		const defaultOptions: InstanceOptions = {
```

Comment:
Maybe we could extract the default options (plus some validation that's happening here) outside the command itself.